markdown/dev/reference/api/config/options/const/en.md

```diff
@@ -26,12 +26,11 @@ options: {
 
 There are typically two use-cases for constant options:
 
-- Rather than define constants in your code, it's good practice to set 
-them in your configuration file.  This way, people who extend your 
-pattern can change them if they would like to.
-- A constant option can be used as a feature-flag. Enabling or disabling
-parts of the code beyond the control of the end user, but accessible to
-developers.
+-   Rather than define constants in your code, it's good practice to set
```

Comment:
Not sure if this multi line bullet point is rendering correctly, may be worth checking

Reply:
It renders correctly.